mintpy/utils/plot.py

```diff
@@ -24,7 +24,10 @@
 )
 from matplotlib.colors import LinearSegmentedColormap
 from mpl_toolkits.axes_grid1 import make_axes_locatable
-from mpl_toolkits.basemap import Basemap, pyproj
+
+from cartopy import crs as ccrs, geodesic as cgeo
+from cartopy.mpl import geoaxes, gridliner
+import shapely.geometry as sgeom
```

Comment:
Since the `shapely` module is used only once, is it possible for us to replace it with our own code, to avoid adding more dependency?

Reply:
Its currently being used to calculate the width of the dataset in kilometers. Do you have an easy way to do this without Basemap? The one we discussed earlier is reliant on Basemap and so won't work.

Reply:
Hi @Ovec8hkin, the following should work. `pyproj` is independent of basemap, even though it can be imported through basemap too.

```
import pyproj
geod = pyproj.Geod(ellps='WGS84')
dist = geod.inv(lon1, lat1, lon2, lat2)[2]
```